testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeColumnMappingMode.java

```diff
@@ -139,6 +139,79 @@ public void testTrinoColumnMappingModeReaderAndWriterVersion(String mode)
                 ")"));
     }
 
+    @Test(groups = {DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS}, dataProvider = "columnMappingDataProvider")
+    public void testChangingColumnMappingModeViaCreateOrReplaceTableOnTrino(String mode)
+    {
+        String tableName = "test_cortas_column_mapping_" + randomNameSuffix();
+        onTrino().executeQuery("" +
+                "CREATE TABLE delta.default." + tableName +
+                "(x INT) " +
+                "WITH (" +
+                " location = 's3://" + bucketName + "/databricks-compatibility-test-" + tableName + "'" +
+                ")");
+
+        assertTableReaderAndWriterVersion("default", tableName, "1", "2");
+
+        // Replace table with a different column mode
+        onTrino().executeQuery("" +
+                "CREATE OR REPLACE TABLE delta.default." + tableName +
+                "(x INT) " +
+                "WITH (" +
+                " location = 's3://" + bucketName + "/databricks-compatibility-test-" + tableName + "'," +
+                " column_mapping_mode = '" + mode + "'" +
+                ")");
+
+        assertTableReaderAndWriterVersion("default", tableName, "2", "5");
+
+        // Revert back to `none` column mode
+        onTrino().executeQuery("" +
+                "CREATE OR REPLACE TABLE delta.default." + tableName +
+                "(x INT) " +
+                "WITH (" +
+                " location = 's3://" + bucketName + "/databricks-compatibility-test-" + tableName + "'," +
+                " column_mapping_mode = 'none'" +
+                ")");
+
+        assertTableReaderAndWriterVersion("default", tableName, "2", "5");
+
+        onTrino().executeQuery("DROP TABLE delta.default." + tableName);
+    }
+
+    @Test(groups = {DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS}, dataProvider = "columnMappingDataProvider")
+    public void testChangingColumnMappingModeViaCreateOrReplaceTableOnDelta(String mode)
```

Comment:
Can we have a similar integration test in `trino-delta-lake`

- create table with column mapping none
- create or replace table (CORTAS) with column mapping name/id
- see if the content is correctly read in the new version of the table